src/components/node-list/node-list.test.js

```diff
@@ -86,7 +88,9 @@ describe('NodeList', () => {
     });
   });
 
-  describe('visibility checkboxes on element items', () => {
+  // these tests refers to search and linked behaviour on the filter panel, which will be updated once the filter panel is merged
+  // into this PR
+  describe.skip('visibility checkboxes on element items', () => {
```

Comment:
same as above

Reply:
the above test is now resolved in my latest commit with modification to match the latest tree-search UI